src/linear_algebra/span.lean

```diff
@@ -96,6 +96,13 @@ by { rw [span_le, comap_coe], exact preimage_mono (subset_span), }
 
 alias submodule.span_preimage_le ← _root_.linear_map.span_preimage_le
 
+lemma closure_subset_span {s : set M} :
+  (add_submonoid.closure s : set M) ⊆ span R s :=
+(@add_submonoid.closure_le _ _ _ (span R s).to_add_submonoid).mpr subset_span
```

Comment:
Can you add the version that says `(add_submonoid.closure s : set M) ≤ (span R s).to_add_submonoid` too?

Reply:
Just this comment left.